lib/src/rewrite.rs

```diff
@@ -776,6 +780,7 @@ pub struct DuplicateCommitsStats {
 pub fn duplicate_commits(
     mut_repo: &mut MutableRepo,
     target_commits: &[CommitId],
+    target_descriptions: &HashMap<CommitId, String>,
```

Comment:
I chose not to make this `Into<String>` because you then have to use turbofishes for `HashMap::new()` (that is, the empty case).